modules/API/API.js

```diff
@@ -198,6 +213,9 @@ function initCommands() {
 
             APP.store.dispatch(reject(participantId));
         },
+        'remove-breakout-room': breakoutRoomJid => {
```

Comment:
..but given that the UI already checks for moderator rights, it's only expected to have the API checking for them too.

Reply:
I haven't been inquired :-) The backend makes those operations safe, as in, they will be ignored if not sent by a moderator.

That said, it would be nice to give the user an error if they try to use these APIs wile not being a moderator.

Reply:
@saghul Done. Should I cover all the other commands in that regard or should I leave it for another PR?